DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -175,6 +175,14 @@ UserAccountSet accounts
                 IsPersistent = rememberMe,
                 IssuedUtc = DateTime.UtcNow
             };
+
+            //HEEDLS-673
+            //Curse progress doesn't get updated if the auth token gets expires by the end of the tutorial so it's set to max 8 hours

```

Comment:
Nice, just a couple of tweaks would be good. 
* Space at the start of the comment
* Full stop at the end
* Typo in course
* Rather than saying "so it's set to max 8 hours" can we say that some tutorials are longer than the default auth token lifetime.
* Can we give a bit more detail about how HEEDLS-673 relates to this, so it doesn't get mistaken for todo or something else. Just say "see HEEDLS-637 for more context" or something like that.